udf/combine_adjacent_days_28_bits.sql

```diff
@@ -1,4 +1,18 @@
-CREATE OR REPLACE FUNCTION
-  udf.combine_adjacent_days_28_bits(prev INT64,
-    curr INT64) AS (udf.shift_28_bits_one_day(prev) + IFNULL(curr,
-    0));
+/*
+
+Combines two bit patterns. The first pattern represents activity over a 28-day
+period ending "yesterday". The second pattern represents activity as observed
+today (usually just 0 or 1). We shift the bits in the first pattern by one to
+set the new baseline as "today", then perform a bitwise OR of the two patterns.
+
+*/
+CREATE OR REPLACE FUNCTION udf.combine_adjacent_days_28_bits(prev INT64, curr INT64) AS (
+  udf.shift_28_bits_one_day(prev) | IFNULL(curr, 0)
```

Comment:
Updated this combine_adjacent_days_28_bits function to use `|` instead of `+`. They end up doing the same thing in the simple case where `curr` is either 0 or 1, but the logical OR `|` is the correct thing to use now that we can have `curr` values that reflect activity on previous days.